tests/tests_cluster_basic.yml

```diff
@@ -0,0 +1,73 @@
+- name: Minimal cluster configuration
+  hosts: all
+  vars_files: vars.yml
```

Comment:
please move this into a `tests/vars/` subdirectory

Reply:
fixed in 1c2a9bf